index.js

```diff
@@ -1,6 +1,55 @@
 /* jshint node: true */
 'use strict';
 
+const CSStoJSON = require('./lib/css-plugin/index');
+const BabelPlugin = require('./lib/babel-plugin/index');
+const MergeTrees = require('broccoli-merge-trees');
+const Funnel = require('broccoli-funnel');
+
 module.exports = {
-  name: 'ember-css-composer'
+  name: 'ember-css-composer',
+  
+  included: function(app) {
+    this._super.included(app);
+
+    app.options = app.options || {};
+    app.options.babel = app.options.babel || {};
+    app.options.babel.plugins = app.options.babel.plugins || [];
+
+    if (!this._registeredWithBabel) {
+      // TODO: Only need to transform files in the app & addon. Maybe
+      // make this a preprocessor with broccoli-funnel
+      app.options.babel.plugins.push(BabelPlugin);
```

Comment:
I made a note here — but, by registering this as a Babel Plugin, it makes initial build times _really_ slow. It should be pretty straightforward to filter out _which_ files are available for processing (it really should _only_ be the files in the app that is directly consuming this addon).